api/src/Service/EventService.php

```diff
@@ -68,7 +75,17 @@ public function listAll(): array
 
     public function getById(/*.mixed.*/$id): array
     {
-        throw new Exception(__CLASS__.": Method '__FUNCTION__' not implemented");
+        $data = $this->eventRepository->selectById($id);
```

Comment:
Don't need to change it here, but I'm starting to realize why ORMs split out the get one item vs get many items.